packages/ui/src/source-media-grid.tsx

```
@@ -421,17 +464,195 @@ export function SourceMediaGrid(props: SourceMediaGridProps) {
+				{ align: "auto" },
+			);
+		}
+		if (focusFrameId !== undefined) cancelAnimationFrame(focusFrameId);
+		focusRenderedMedia(nextMedia.id);
+	};
+
 	const gridContent = (
 		<div
 			class="@container relative min-w-0 w-full"
+			aria-label="メディア一覧"
+			role="grid"
```

Biome (2.5.6)

[error] The elements with this role can be changed to the following elements:
<table>

(lint/a11y/useSemanticElements)